Cargo.toml

```diff
@@ -7,3 +7,68 @@ members = [
 
 [profile.release]
 panic = "unwind"
+
+[patch."https://github.com/paritytech/substrate"]
```

Comment:
Wait why do we need this patch ?

Reply:
The only reason I have to do so is that we included a pallet `pallet-collator-selection`, but whose deps are from substrate's master branch.
But now, we have to bump all deps to `polkadot-v0.9.5`, it will cause compilation problem.
For example, `manta-xassets is using `sp-runtime` that is from `polkadot-v0.9.5`,
but if `pallet-collator-selection` uses sp-runtime` that is from `master`, now there're two version of `sp-runtime` in Cargo.lock. Then compilation will complain.

Reply:
Thanks. Maybe add this as a comment.